src/control/ZoomPanHandler.ts

```diff
@@ -231,7 +234,7 @@ export class ZoomPanHandler {
    * Initialized handler
    */
   init(): void {
-    this.zoom = zoom().scaleExtent([0.1, this.options.maxZoomLevel]).on('zoom', this.onZoom);
+    this.zoom = zoom().scaleExtent([DEFAULT_MIN_ZOOM_LEVEL, this.options.maxZoomLevel]).on('zoom', this.onZoom);
```

Comment:
Maybe min zoom level should be possible to set through ZoomAndPanOptions as well?

Reply:
done